installer/roles/ansible-service-broker-setup/files/provision-ansible-service-broker.sh

```diff
@@ -1,13 +1,14 @@
 #!/bin/bash
 set -e
 
-readonly DOCKERHUB_USER="${1}"
-readonly DOCKERHUB_PASS="${2}"
-readonly DOCKERHUB_ORG="${3}"
-readonly LAUNCH_APB_ON_BIND="${4}"
-readonly TAG="${5}"
-readonly WILDCARD_DNS="${6}"
-readonly ANSIBLE_SERVICE_BROKER_NAMESPACE="${7}"
+readonly DOCKERHUB_USER=${1:?"[ERROR]You must provide a dockerhub username."}
```

Comment:
@vchepeli I'm hitting an issue with local provsion with this change.

My dockerhub password has a symbol `)` in it, which doesn't get parsed OK.
```
"stderr_lines": ["/bin/sh: -c: line 0: syntax error near unexpected token `)'"
```

Can we add back quotes around this?

Reply:
Added already. @david-martin can you try it again please?